src/components/AddressSearch.js

```diff
@@ -278,8 +279,8 @@ const AddressSearch = (props) => {
                     onLayout={(event) => {
                         // We use the height of the element to determine if we should hide the border of the listView dropdown
                         // to prevent a lingering border when there are no address suggestions.
-                        // The height of the empty element is 2px (1px height for each top and bottom borders)
-                        setDisplayListViewBorder(event.nativeEvent.layout.height > 2);
+                        // The height of the empty element is 14px (2px height for borders and 12px height for vertical padding)
+                        setDisplayListViewBorder(event.nativeEvent.layout.height > variables.googleEmptyListViewHeight);
```

Comment:
I don't think we need a variable here. But please mention that those values are coming from `getGoolgeListViewStyle`

Reply:
Agree. Actually, I think the 12 padding is the one that we should put in a variable because we use it on several popover too.

Btw, we should also fix the `goolge` typo here 🤣

Reply:
For vertical padding we don't have one for 6 (12 total) https://github.com/Expensify/App/blob/4947508bedc2e901082d805c84498f5f394ef655/src/styles/utilities/spacing.js#L265-L275